docs/Reference/API-Methods.md

```diff
@@ -360,28 +362,34 @@ Properties of the remote node object are:
 
         ```json
         {
-          "jsonrpc" : "2.0",
-          "id" : 1,
-          "result" : [
-            {
-              "version": "0x5",
-              "name": "Parity-Ethereum/v2.3.0-nightly-1c2e121-20181116/x86_64-linux-gnu/rustc1.30.0",
-              "caps": [
+           "jsonrpc": "2.0",
+           "id": 1,
+           "result": [
+             {
+               "version": "0x5",
+               "name": "besu/v20.10.4-dev-0905d1b2/osx-x86_64/adoptopenjdk-java-11",
+               "caps": [
                  "eth/62",
                  "eth/63",
-                 "par/1",
-                 "par/2",
-                 "par/3",
-                 "pip/1"
-              ],
+                 "eth/64",
+                 "eth/65",
+                 "IBF/1"
+               ],
                "network": {
-                  "localAddress": "192.168.1.229:50115",
-                  "remoteAddress": "168.61.153.255:40303"
+                 "localAddress": "127.0.0.1:63139",
```

Comment:
not sure we should change the addresses to localhost. The previous example was better to help understand what you can get.

Reply:
Updated